connectors/src/connectors/google_drive/temporal/activities.ts

```diff
@@ -685,7 +686,7 @@ export async function renewWebhooks(pageSize: number): Promise<number> {
       const res = await ensureWebhookForDriveId(
         connector,
         wh.driveId,
-        GOOGLE_DRIVE_WEBHOOK_RENEW_MARGIN_MS
+        renewWebhookStartingTime
```

Comment:
The implementation of this function used to do `new Date(new Date().getTime() + marginMs)`

But now you're passing `renewWebhookStartingTime` hiwch is `new Date().getTime();` ?

Reply:
Indeed, let me rename the function parameter name for better clarity though

Reply:
This way we make sure that there's no race condition